### ISSUE TITLE
Add appropriate keyboard interactions for collapse links, buttons

### ISSUE THREAD

**Issue by cjg89** [UCF/Athena-Framework#173]
**Description**

Incorporate appropriate keyboard interactions for collapse toggles.  Accordions should follow the WAI ARIA spec for an accordion component (https://www.w3.org/TR/wai-aria-practices-1.1/#accordion), and basic collapse toggle buttons/links should probably follow the spec for "disclosures": https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure 

**Why it's Important**

Currently, collapse toggles cannot be toggled as one would expect to via keyboard.

**Alternatives**

n/a

**Possible Implementation**

Add keyboard interactions as needed, and update the last paragraph of the docs here: https://ucf.github.io/Athena-Framework/components/collapse/#accessibility

**Additional context**

n/a

**Comment by jmbarne3** [comment]
Solved in #175
